src/components/screens/home-screen/home-screen.tsx

```diff
@@ -119,6 +125,20 @@ class HomeScreen extends React.Component<Props, State> {
     );
   }
 
+  /**
+   * Method for greeting message
+   */
+  private greeting = () =>{
+    const hour = new Date().getHours();
```

Comment:
```suggestion
    const { moriningHour, noonHour, eveningHour } = this.state;
    
    const hour = new Date().getHours();
```
It is good to assign consts like this. This allows to use i.e morning hour instead of this.state.morningHour